examples/index.html

```diff
@@ -6,6 +6,9 @@
 </head>
 <body>
   <custom-element></custom-element>
-  <script type="module" src="../index.js"></script>
-</body>
+  
+  <!-- Development script, uncomment when running locally and comment out the production one -->
+  <!-- <script src="../dist/index.umd.js"></script> -->
+  <!-- Production script -->
+  <script src="https://unpkg.com/@github/auto-complete-element@latest/dist/index.umd.js"></script></body>
```

Comment:
For find and replace:

```suggestion
  <script src="https://unpkg.com/@github/custom-element-boilerplate@latest/dist/index.umd.js"></script></body>
```

Reply:
Ah yes 😬 !